python/src/GeneralLinearModelAlgorithm_doc.i.in

```diff
@@ -55,23 +62,25 @@ We note:
     \vect{\beta}^\ell = \left(
       \begin{array}{l}
         \beta_1^\ell \\
-        \dots  \\
+        \vdots  \\
         \beta_{n_\ell}^\ell
        \end{array}
      \right) \in \Rset^{n_\ell}
      \quad \mbox{ and } \quad
      \vect{\beta} = \left(
       \begin{array}{l}
          \vect{\beta}^1\\
-        \dots  \\
-         \vect{\beta}^d
+         \vdots  \\
+         \vect{\beta}^\outputDim
        \end{array}
      \right)\in \Rset^{\sum_{\ell=1}^p n_\ell}
 
 
-The *GeneralLinearModelAlgorithm* class estimates the coefficients :math:`\beta_j^\ell` and :math:`\vect{p}` where :math:`\vect{p}` is the vector of parameters of the covariance model (a subset of :math:`\vect{\theta}, \vect{\sigma}, \mat{R}, \vect{\lambda}`) that has been declared as *active* (by default, the full vectors :math:`\vect{\theta}` and :math:`\vect{\sigma}`).
-
-The estimation is done by maximizing the *reduced* log-likelihood of the model, see its expression below.
+The *GeneralLinearModelAlgorithm* class estimates the coefficients :math:`\beta_j^\ell` and :math:`\vect{p}`
+where :math:`\vect{p}` is the vector of parameters of the covariance model (a subset of
+:math:`\vect{\theta}, \vect{\sigma}, \mat{R}, \vect{\lambda}`) that has been declared as
+*active* (by default, the full vectors :math:`\vect{\theta}` and :math:`\vect{\sigma}`).
+The estimation is done by maximizing the *reduced* log-likelihood of the model (see its expression below).
```

Comment:
Where is below? Maybe give the equation number?

Reply:
Fixed at 2484465a1a62fa5784f2e76ce1411e17f892bf0e